config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/dishes/:id", to: "dishes#show"
+  patch "/dishes/:id", to: "dish_ingredients#new"
```

Comment:
A more RESTful option for this path to "dish_ingredients#create" could be POST to "/dishes/:id/ingredients" because you are not updating the attribute of a dish but rather adding an association between 2 resources.